mozci/push.py

```diff
@@ -882,11 +882,26 @@ def get_candidate_regressions(
         associated task failed (therefore including intermittents), and which
         is either not classified or fixed by commit.
 
+        In case the `build` parameter is set to True, only return tasks that have a build
+        type and match specific checks.
+
         Returns:
             set: Set of runnable names (str).
         """
         logger.debug(f"Retrieving candidate regressions for {self.rev}...")
 
+        if build:
+            if runnable_type != "label":
+                raise ValueError(
+                    "Only label can be set as runnable type to analyze build regressions"
+                )
+            return {
+                task.label: (0.0, summary)
+                for task in self.tasks
+                if self._check_build_task_regression(task)
+                and (summary := self.label_summaries.get(task.label))
+            }
```

Comment:
Question: if we don't add this, doesn't `get_candidate_regressions` already returns builds?

Reply:
Yes, by default `get_candidate_regressions` will return 61 results that include the 7 returned by `_should_retrigger_build`.